xarray/backends/netCDF4_.py

```diff
@@ -584,6 +591,7 @@ def open_dataset(
             persist=persist,
             lock=lock,
             autoclose=autoclose,
+            memory=memory,
```

Comment:
Also here type hints would be appreciated!

Reply:
I don't see how I could add type hints here, exactly.

Reply:
Sry, wrong line. Ignore this, haha.